src/pretix/control/templates/pretixcontrol/event/settings.html

```diff
@@ -83,12 +83,16 @@ <h1>{% trans "General settings" %}</h1>
             <fieldset>
                 <legend>{% trans "Attendee data" %}</legend>
                 {% bootstrap_field sform.attendee_names_asked layout="control" %}
-                {% bootstrap_field sform.attendee_names_required layout="control" %}
+                {% bootstrap_field sform.attendee_names_required lddressformayout="control" %}
```

Comment:
```suggestion
                {% bootstrap_field sform.attendee_names_required layout="control" %}
```

I guess, the `ddressform` (`addressform`?)-leftover is not needed anymore at all...